APACHE-LICENSE-2.0.txt

```diff
@@ -0,0 +1,202 @@
+
```

Comment:
I don't think this is good in the root folder ... it might confuse people about the license.

Reply:
done